### PR TITLE
[WIP] New page about CMT

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Better content (waiting for it)

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
We need a better image for kpp

File: source/convenant-medische-technologie.erb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <div class="hero-copy">
+      <h1>Het Convenant Medische Technologie borgen met het Kwaliteitspaspoort&trade;</h1>
+  </div>
+</div>
+
+<section class="intro">
+  <div class="row">
+    <article>
+      <p>
+<<<<<<< HEAD
```
Comment (dkhgh) [inline source/convenant-medische-technologie.erb]: Looks like a rebase artifact

File: source/stylesheets/_convenant.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      rgba($blue, 0.5) 33%,
+      rgba($blue, 0.5) 66%,
+      $blue-alt 66%,
+      $fallback: $blue-alt);
+
+    background-attachment: scroll;
+
+    .hero-inner {
+<<<<<<< HEAD
+      padding: 3em 1em 0em;
```
Comment (dkhgh) [inline source/stylesheets/_convenant.scss]: Something something rebase

File: source/blog/2016-01-08-the-future-is-now.html.markdown [hunk 3]
```diff
@@ -15,6 +15,4 @@ Visies kunnen vluchtig zijn. Als morgen een nieuwe visionair opstaat kan een oud
 
 Daarom wil ik het in 2016 graag met je hebben over leren en ontwikkelen in het ziekenhuis van 2016. We gaan dit jaar aan de slag, praktisch en concreet. Op basis van ervaringen, praktijkcases en inzichten uit wetenschap *(en natuurlijk ook de inspiratie van visionairs!)* wil ik met u toewerken naar een **haalbare kwaliteitsslag** in leren en ontwikkelen. Vandaag uitvoerbaar, maar flexibel genoeg om ook door te groeien naar 20XX. Ga je met mij deze uitdaging aan?
 
-### The future is now, let’s create it.
```
Comment (fatboypunk) [inline source/blog/2016-01-08-the-future-is-now.html.markdown]: This line is already removed on master right?

**Pull request merged**